docs/knowledge_base/threat_feed.md

```diff
@@ -97,3 +99,51 @@ For example, if you want to limit the items per page to 3 since July 19, 2023 yo
 ```text
 https://threats.phylum.io/?per_page=3&since=2023-07-19
 ```
+
+## Cursor Based Pagination
+
+The threat feed supports cursor based pagination. By providing the `cursor` parameter you can iterate through the data in a guaranteed order. For example:
+
+```bash
+curl "https://threats.phylum.io/?cursor=84" -H "Authorization: Bearer $PHYLUM_API"
+```
+
+If additional rows exist, the response will include the next cursor in the `cursor` key. The cursor value will be `null` when no additional rows exist.
+
+```json
+{
+  "cursor": 2722653,
+  "packages": [
+    {
+      "created": "Wed, 28 Aug 2024 22:24:28 GMT",
+      "ecosystem": "npm",
+      "hashes": null,
+      "indicators": {
+        "npm_security_holding_rule": true,
+        "triaged_malware_rule": true
+      },
+      "name": "@hishprorg/sequi-perspiciatis",
+      "version": "0.0.1-security"
+    },
+    {
+      "created": "Wed, 28 Aug 2024 23:29:13 GMT",
+      "ecosystem": "npm",
+      "hashes": null,
+      "indicators": {
+        "npm_security_holding_rule": true,
+        "triaged_malware_rule": true
+      },
+      "name": "@hishprorg/necessitatibus-minus-quos",
+      "version": "0.0.1-security"
+    }
+  ]
+}
+```
+
+## Additional Query Options
```

Comment:
```suggestion
## Additional Cursor Query Options

```
If this heading is going to be used, it should:

* Have a blank line after it
* Be different than other headings in this document
  * See [markdownlint rule 024](https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md024.md) for full detail, but basically it makes it harder to create anchor links here
  * The suggested wording change here is just one option

Another option is to remove this heading entirely.

Reply:
Removed the heading entirely, there isn't a tables worth of stuff like above so it seemed superfluous here.